app/styles/main.css

```diff
@@ -197,3 +197,11 @@ body {
   text-overflow: ellipsis;
   margin-bottom: 20px;
 }
+
+label,input {
+display: block;
+}
+
+label{
+margin-bottom: 10px;
```

Comment:
move this as well after body like what my comment above explained.

Also, be mindful of the spacing. Basically want the indentation to line up like everything else in the css file

label {
    margin-bottom: 10px;
}